--- pluto-0.11.4/configure.ac.orig	2015-10-28 16:24:59.000000000 +0100
+++ pluto-0.11.4/configure.ac	2015-12-03 16:02:55.363732747 +0100
@@ -114,6 +114,7 @@
 
 
 
+if test x$external_piplib = xfalse ; then
 dnl Configuring PipLib (long long int is 64 bits)
 echo ""
 echo "=========================="
@@ -126,8 +127,10 @@
  $RM config.cache;
  ./configure ${configureopts} ${archopts}
  )
+fi
 
 
+if test x$external_openscop = xfalse ; then
 dnl Configuring Openscop
 echo ""
 echo "=========================="
@@ -139,8 +142,10 @@
 (cd openscop/
  ./configure ${configureopts} ${archopts}
  )
+fi
 
 
+if test x$external_clan = xfalse ; then
 dnl Configuring Clan-osl
 echo ""
 echo "=========================="
@@ -155,8 +160,10 @@
 
 dnl check if clan configure failed
 AC_CHECK_FILE([clan/Makefile], [], [AC_MSG_ERROR([configure in clan/ failed])])
+fi
 
 
+if test x$external_candl = xfalse ; then
 dnl Configuring Candl-osl
 dnl can't enable the gmp version without modifying
 dnl the precision the is used elsewhere in pluto
@@ -173,8 +180,10 @@
 (cd candl/
  ./configure ${configureopts} ${archopts}
  )
+fi
 
 
+if test x$external_polylib = xfalse ; then
 dnl Configuring polylib
 echo ""
 echo "======================"
@@ -185,6 +194,7 @@
 (cd polylib/
  ./configure ${configureopts}
 )
+fi
 
 
 AC_SUBST(ISL_INCLUDE)
@@ -222,6 +232,7 @@
 fi
 
 
+if test x$external_cloog = xfalse; then
 dnl Configuring Cloog 0.14.1 (long long int)
 echo ""
 echo "=========================="
@@ -247,6 +258,7 @@
 (cd cloog-isl/
  ./configure ${configureopts} ${archopts}
  )
+fi
 
 AC_PATH_PROGS(BASH, bash)
 
--- pluto-0.11.0/Makefile.am.orig	2014-05-30 10:28:06.000000000 +0200
+++ pluto-0.11.0/Makefile.am	2014-10-05 15:40:59.042473155 +0200
@@ -18,7 +18,7 @@
 endif
 
 
-SUBDIRS = piplib openscop clan candl polylib $(MAY_ISL) cloog-isl src
+SUBDIRS = src
 
 ACLOCAL_AMFLAGS = -I m4
 
--- pluto-0.11.4/src/Makefile.am.orig	2015-12-03 15:33:25.990473676 +0100
+++ pluto-0.11.4/src/Makefile.am	2015-12-03 16:12:33.643708512 +0100
@@ -27,25 +27,18 @@
 endif
 
 pluto_CPPFLAGS = $(OPT_FLAGS) $(DEBUG_FLAGS) -funroll-loops -fopenmp \
-   -DSCOPLIB_INT_T_IS_LONGLONG -DCLOOG_INT_GMP \
-   -I../include \
-   -I../piplib/include \
-   -I../clan/include \
-   $(ISL_INCLUDE) \
-   -I../cloog-isl/include \
-   -I../openscop/include \
-   -I../candl/include \
-   -I../polylib/include
+   -DSCOPLIB_INT_T_IS_MP -DCLOOG_INT_GMP \
+   -I../include
 
-pluto_LDFLAGS = -static -fopenmp
+pluto_LDFLAGS = -fopenmp
 pluto_LDADD =  \
-   ../piplib/libpiplib64.la \
-   ../openscop/libosl.la \
-   ../clan/libclan.la \
-   ../cloog-isl/libcloog-isl.la \
-   $(ISL_LIBADD) \
-   ../polylib/libpolylib64.la \
-   ../candl/libcandl.la \
+   -lpiplibMP \
+   -losl \
+   -lclan \
+   -lcloog-isl \
+   -lisl \
+   -lpolylibgmp \
+   -lcandl \
    -lm
 
 lib_LTLIBRARIES = libpluto.la
@@ -53,20 +46,14 @@
 # libpluto doesn't use polylib-based functions
 libpluto_la_SOURCES = constraints.c constraints_isl.c math_support.c ddg.c transforms.c pluto.c post_transform.c framework.c polyloop.c iss.c libpluto.c libpluto_dummy.c constraints.h math_support.h pluto.h program.c program.h tile.c
 libpluto_la_CPPFLAGS = $(OPT_FLAGS) $(DEBUG_FLAGS) \
-   -DLIB_PLUTO -DSCOPLIB_INT_T_IS_LONGLONG -DCLOOG_INT_GMP \
-   -I../include \
-   -I../piplib/include \
-   -I../clan/include \
-   -I../openscop/include \
-   -I../candl/include \
-   -I../cloog-isl/include \
-   $(ISL_INCLUDE)
+   -DLIB_PLUTO -DSCOPLIB_INT_T_IS_MP -DCLOOG_INT_GMP \
+   -I../include
 
 # libpluto doesn't use polylib
 libpluto_la_LIBADD = \
-   ../piplib/libpiplib64.la \
-   ../openscop/libosl.la \
-   ../clan/libclan.la \
-   ../candl/libcandl.la \
-   $(ISL_LIBADD) \
+   -lpiplibMP \
+   -losl \
+   -lclan \
+   -lcandl \
+   -lisl \
    -lm -lgomp
