diff -urNp -x '*.orig' pluto-0.11.4.org/include/pluto/libpluto.h pluto-0.11.4/include/pluto/libpluto.h
--- pluto-0.11.4.org/include/pluto/libpluto.h	2015-10-28 16:24:59.000000000 +0100
+++ pluto-0.11.4/include/pluto/libpluto.h	2022-02-16 09:51:43.867465467 +0100
@@ -1,7 +1,8 @@
 #ifndef __LIBPLUTO__
 #define __LIBPLUTO__
-#include "isl/union_set.h"
-#include "isl/union_map.h"
+#include <isl/union_set.h>
+#include <isl/union_map.h>
+#include <osl/scop.h>
 
 #include "osl/scop.h"
 
